### DISCUSSION TITLE
redis做了cluster集群 配置和代码需要什么改动？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#242] [02.提问]
redis做了cluster集群 配置和代码需要什么改动？

**Comment by zuihou** [comment]
lamp-cloud 只对 redisTemplate 多封装了一层 CacheOps， 目的是方便调用，方便定制Cache Key。


至于集群如何配置，请参看 spring boot 配置 redis 集群。

**Reply by zqb666** [reply]
spring boot 配置 redis 集群  这个文档没找到 在哪里？

**Reply by zuihou** [reply]
https://www.baidu.com/s?tn=28114124_2_dg&wd=spring%20boot%20redis%20%E9%9B%86%E7%BE%A4

**Reply by zqb666** [reply]
我还以为是 现成的文档

**Reply by zqb666** [reply]
项目中 RedisOps类 redis改成 cluster架构后需要变动代码吗

**Reply by zuihou** [reply]
RedisOps 实际上就是对 redisTemplate 的 封装，redisTemplate就是spring 官方对redis 的封装， 单机还是集群不影响redisTemplate，故不影响 RedisOps
